frontend/apps/zk-poker/src/features/signless/provider.tsx

```diff
@@ -0,0 +1,142 @@
+import { EnableSessionModal } from '@ez/features/signless-transactions/components/enable-session-modal';
```

Comment:
Let's discuss the necessity of having an auto-sign feature as a part of `gear-ez-transactions`. I would like to see a standalone PR with changes related to it, or now it's hard to review the code related to it since the PR is quite big.

I like modular changes that you implemented without affecting the main package, but there's some level of the complexity that was added, I would like to investigate whether it can be simplified.